curator/views/study/edit.html

```diff
@@ -2089,6 +2089,7 @@ <h3 id='dialog-heading'>
 </div>
 
 <!-- hidden template for gathering comments when saving the study -->
+<!-- this version when we have user email -->
```

Comment:
Fixed with [this commit](https://github.com/OpenTreeOfLife/opentree/pull/972/commits/9281c7efc52479ff9912091a7717c4699209220c)